amplipi/streams.py

```diff
@@ -1152,15 +1152,15 @@ def reconfig(self, **kwargs):
       if self._is_running():
         self.reactivate()
 
-  def connect(self, src):
+  def _activate(self, vsrc: int):
     """ Connect a squeezelite output to a given audio source
     This will create a LMS client based on the given name
     """
     f = open(f"lms_{str(self.name).replace(' ', '_')}_metadata.json", "w", encoding="UTF-8")
     json.dump(self.meta, f, indent = 2)
     f.close()
     if self.mock:
-      self._connect(src)
+      self._connect(vsrc)
```

Comment:
Please revert this change, connect should not be called directly by a PersistantStream.